optaplanner-website-root/content/blog/2023/04/05/BoostVRPListSelectorScalability.adoc

```diff
@@ -0,0 +1,102 @@
+= Boosting list variable scalability with nearby selection
+dupliaka
+2023-04-05
```

Comment:
Let's not forget to update the date as well as the directory structure when the blogpost is ready for publishing.